packaging/package/jar_packager.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+from connector_file import ConnectorFile
+from zipfile import ZipFile
```

Comment:
Did you have to pip install any of these modules? If so, it needs to be added to setup.y's install field.

Reply:
`os`, `zipfile`, and `pathlib` are part of the standard library, so they're gtg 👍 

I would suggest reordering the imports [per PEP-8](https://www.python.org/dev/peps/pep-0008/#imports): standard library, 3rd party, local applications.

```python
import os
from pathlib import PathLib
from zipfile import ZipFile

from connector_file import ConnectorFile
from manifest import Manifest
```

Reply:
ok I will merge this PR and send another PR about those two issues, since it takes a while to revert remote branch and recommit the whole change.